configure/cmake_win.cmake

```diff
@@ -162,8 +162,8 @@ install(DIRECTORY $ENV{OMNI_BASE}/include/omniVms DESTINATION include COMPONENT
 install(FILES $ENV{OMNI_BASE}/include/omniconfig.h DESTINATION include COMPONENT headers)
 install(FILES $ENV{OMNI_BASE}/include/omnithread.h DESTINATION include COMPONENT headers)
 install(FILES $ENV{ZMQ_BASE}/include/zmq.h DESTINATION include COMPONENT headers)
-install(FILES $ENV{ZMQ_BASE}/include/zmq.hpp DESTINATION include COMPONENT headers)
-install(FILES $ENV{ZMQ_BASE}/include/zmq_addon.hpp DESTINATION include COMPONENT headers)
+install(FILES $ENV{CPPZMQ_BASE}/include/zmq.hpp DESTINATION include COMPONENT headers)
```

Comment:
I like the idea of separating zmq and cppzmq, but wouldn't this break other people's builds if they do not set CPPZMQ_BASE? Would it be possible to fall-back to using ZMQ_BASE if CPPZMQ_BASE is not defined? - just a suggestion, I have no experience in windows builds and do not know if this will be a problem.

Reply:
Yes. And yes, it is possible. IMO, let them break. This was broken to begin with, and the CMake system is not very old yet. To be honest, the whole build system right now is super unportable and seems to be tailored towards one specific build scenario only.
The problem of _not being able to build the software_ is better remedied with a **manual** or an example. This example can be the appveyor build, or if you want, my conan build. You should not already stack compatibility code on this to keep a broken solution running.

Reply:
Fine with me, but let's wait for comments from people using windows.

Reply:
> Yes. And yes, it is possible. IMO, let them break. This was broken to begin with, and the CMake system is not very old yet. To be honest, the whole build system right now is super unportable and seems to be tailored towards one specific build scenario only.

You're right, it was tailored for appveyor builds with the original goal to generate some zip files and Windows installers containing the Tango library and include files and all the dependencies (lib and include files too).
I think if we really want to help our users to use CMake to build cppTango themselves on Windows, then we need to change several things.
One of them being to stop installing the dependencies in the cppTango CMakefiles as you suggested.
On Linux, we don't provide any longer zmq*.hpp files for instance. We should probably do the same on Windows by default too. This could be an option?
As you suggested, @ltjax, this dependencies installation could be moved to a separate CMake script, for instance.

Using conan might also simplify our life... I'm not familiar with it yet.
Appveyor could also probably be updated to use conan and to create conan packages, I guess you would be interested in that?

It would be interesting to get some feedback from our users on Windows.

Reply:
Thanks for the explanation!
Conan really streamlines dependency handling, and I recommend getting into it.
Yes, I am interested in getting binary deployment like that working on appveyor, but I fear I might need a little more budget from the guys at KARA/ANKA (or you?) to do that. For now, my focus is to get one old windows tango device up and running again, and the working recipe is almost as good for that as a full binary release.